app/data/tasks/fetch_record_csv.py

```diff
@@ -37,11 +37,22 @@ def date_iso(d):
 
 
 @shared_task
-def export_records(occurred_min, occurred_max):
+def export_records(occurred_min, occurred_max, record_label=settings.PRIMARY_RECORD_LABEL):
+    def to_utf8(s):
+        """Convert to utf8 encoding and strip special whitespace/commas for csv writing"""
+        if isinstance(s, str):
+            return re.sub(r'[\r\n\t,]', '', s)
+        elif isinstance(s, unicode):
+            return re.sub(r'[\r\n\t,]', '', s).encode('utf-8')
+        elif s is None:
+            return unicode('').encode('utf-8')
+        else:
+            return re.sub(r'[\r\n\t,]', '', unicode(s)).encode('utf-8')
```

Comment:
Python's CSV writer should handle commas by wrapping them in quotes, which most CSV readers will correctly understand -- were commas causing problems?

Also, it might be helpful to assign the regex to a variable up top, that way if someone needs to edit it they don't have to change three different places.